pkg/cache/sql/informer/listoption_indexer.go

```diff
@@ -51,8 +55,21 @@ const (
 	createFieldsIndexFmt = `CREATE INDEX "%s_%s_index" ON "%s_fields"("%s")`
 
 	failedToGetFromSliceFmt = "[listoption indexer] failed to get subfield [%s] from slice items: %w"
+
+	// 'key' contains values from the column key of table nodes
```

Comment:
Nitpick: why "nodes"?

Reply:
"Nodes" is cruft from earlier code, and I don't remember why I picked it.  I'll deal with that next tomorrow (Friday Nov 8)

Reply:
There's no need for a comment at all -- it was restating the DDL, and went out of date.